bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -198,6 +198,10 @@ impl RoomListItem {
         RUNTIME.block_on(async { self.inner.name().await })
     }
 
+    fn full_room(&self) -> Arc<Room> {
```

Comment:
What do you mean by `full`?

Reply:
It's a legacy term that means: gimme the matrix_sdk_ffi:Room instance. Happy to change to something more meaningful.

Reply:
Ah ok, as a newcomer this feels a bit strange, maybe `room` would be sufficient? (no strong opinion here, so you choose)